### ISSUE TITLE
Simplify first run experience

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/ios#65]
The current first run experience drags on a lot and also gets technical real fast:

1. I need to swipe through the intro slides or skip them
2. I need to think of yet another at-least-4-letters-long password
3. I need to put in my email or a hint, or know to skip it
4. I need to manually tap on »Add your Nextcloud account«
5. I need to manually tap in the first field to start typing

Instead, how about this simpler version:
1. Maximum of 3 intro slides https://github.com/nextcloud/ios/issues/64
2. After that the fields for Server address, Username & Password directly show, with Server address being focused by default and the keyboard being out.

Basically same as in the Android app.

If it’s difficult to disable encryption (see #51), can we at least not have the encryption key passcode and all the setup in the initial run? Like make it an empty passcode and don’t require the email? That’s a really technical and off-putting part of the experience.

cc @nextcloud/ios @nextcloud/designers

**Comment by marinofaggiana** [comment]
@jancborchardt we are working to make optional the crypto system ... see beta 😄

**Comment by jancborchardt** [comment]
Awesome! Can we then also directly reduce the number of intro slides, have the input fields show directly without the extra "Add your Nextcloud account" screen, and make the server address field focused by default? Those are quick wins. :)

cc @anagromataf
